doc/Development-Guide-Rad-Tool.md

```diff
@@ -3,43 +3,46 @@
   
 ### Introduction
 
- In this document, we will introduce **ASP.NET Zero Power Tools** and explain it. This tool is developed to minimize the effort of creating a new entity. It creates all related layers (including UI) by defining an entity.
+ In this document, we will introduce **ASP.NET Zero Power Tools** and explain it. This tool is developed to minimize the effort of creating a new entity. It creates all the related layers (including UI) by defining an entity.
 
 ### Download And Install
 
- If your project version is 5.1.0+, all you have to do is just installing **ASP.NET Zero Power Tools** extension on visual studio, from https://marketplace.visualstudio.com/items?itemName=Volosoft.AspNetZeroPowerTools.
+ If your project is version 5.1.0+, all you have to do is just instal the **ASP.NET Zero Power Tools** extension on Visual Studio from https://marketplace.visualstudio.com/items?itemName=Volosoft.AspNetZeroPowerTools.
 
- If your project version is below 5.1.0, you also have to copy the AspNetZeroRadTool folder to your own project, from a newly downloaded 5.1.0+ project.
+ If your project is below version 5.1.0, you have to copy the AspNetZeroRadTool folder to your own project from a newly downloaded 5.1.0+ project.
```

Comment:
- "If your project `version` is below ~`version`~ `v`5.1.0"
- `also` should not be removed. This step is in addition to the step above.
- "newly downloaded `v`5.1.0+ project."